src/main/webapp/app/exercises/shared/import/exercise-import.component.ts

```diff
@@ -42,15 +49,37 @@ export class TextExerciseImportComponent implements OnInit {
     isCourseFilter = true;
     isExamFilter = true;
 
-    constructor(private pagingService: TextExercisePagingService, private sortService: SortService, private activeModal: NgbActiveModal) {}
+    titleKey: string;
+
+    constructor(private sortService: SortService, private activeModal: NgbActiveModal, private injector: Injector) {}
 
     ngOnInit(): void {
+        if (!this.exerciseType || this.exerciseType === ExerciseType.FILE_UPLOAD) {
```

Comment:
Is there a reason this is not supported for file upload exercises? Either way, could you add a comment and create an issue if necessary?

Reply:
There is no server logic for this (and before this PR there was simply no `FileUploadImportComponent`). I guess the reasoning here is that file upload exercises are barely used on Artemis and it's not worth the work.

Reply:
I added a comment and opend #6048.